### ISSUE TITLE
Could we use LLaVA to extract information from a given photo?

### ISSUE THREAD

**Issue by nonstopnht** [haotian-liu/LLaVA#93]
Hi LLaVA team, thanks for the great work, it is really impressive of what you have done!
I am trying to understand what we could do with LLaVA, and to extract information from a photo but the answer is a bit interesting. 

While LLaVA is able to tell the uploaded photo is a certificate of incorporation, but it is telling a wrong UEN, is it due to the training is not enough or the limitation of transformer? Thank you for your great effort!

![image](https://user-images.githubusercontent.com/17609528/236123433-9d2c7de6-8bbd-4244-b4ea-6a85ae19fa1b.png)
![image](https://user-images.githubusercontent.com/17609528/236123633-dd58f7cf-e413-402e-81db-c64da568e478.png)
![Screenshot 2023-05-04 135147](https://user-images.githubusercontent.com/17609528/236123649-13df1e3e-39d8-4940-b096-7fb98761104b.png)

**Comment by haotian-liu** [comment]
Hi, thank you for your interest in our work.  The current LLaVA is not good at parsing the details of high resolution images, as the current input resolution is only 224x224.  It is also partially due to that these OCR tasks is not very common in our dataset. We are working on these improvements, stay tuned!

**Comment by nonstopnht** [comment]
Thank you very much for your reply! Looking forward to seeing it, let me know if I can contribute. (I am new to AI, but very interested in it)

**Comment by haotian-liu** [comment]
@nonstopnht Of course, contributions are always welcomed!  If you find anything that you are interested in improving, feel free to let me know :) Thanks!
